src/components/AddItem/AddItem.js

```diff
@@ -74,41 +79,44 @@ const AddItem = () => {
           onChange={onGroceryItemInputChange}
           required
```

Comment:
Did we want to put a min/max for characters on input? I think we discuss 3 min/12 max as kind of a simple "error" handler for breakpoints.